README.md

```diff
@@ -320,6 +320,13 @@ Since v0.7.0, this library supports blockwise transfers, you can trigger
 them by adding a `req.setOption('Block2', Buffer.of(0x2))` to the
 output of [request](#request).
 
+And since v?.?.?, this library supports rudimentry type 1 blockwise transfers, you can trigger
```

Comment:
Needs to be updated after release

Reply:
I assume the collaborators will handle that or would you like me to put in a version?

Reply:
Yes, was mainly a "Note to ourself" :-)

Reply:
@JKRhb can you take care about that after merge and in preparation of release then please?

Reply:
Sure, will do :)